lib/testing/mockRes.json

```diff
@@ -335,7 +385,7 @@
                                     "READ_ACP": []
                                 },
                                 "key": "",
-                                "location": "",
+                                "location": [],
```

Comment:
Not sure, but basing off what I'm seeing in mongo, but location seems to be set to null if none are defined. I'm not sure if an empty array is also set for this if none are defined

Reply:
Right now https://github.com/scality/backbeat/blob/development/8.1/lib/clients/backbeat-2017-07-01.api.json has it defined as a list, so even if a null/empty string is passed the BackbeatClient will format the value to an empty list. it's possible the backbeat client api.json file has an error, and the location value is actually a string ?

Reply:
ah cool beans, ya you're right, if null/empty, I think should just return an empty list, which seems ok to me. The location value seems to be an array though, based on https://github.com/scality/Arsenal/blob/development/8.1/lib/models/ObjectMD.js#L548 so seems good to me 👍